mb_bulk_copy_work_codes.user.js

```diff
@@ -699,8 +695,73 @@ function handleISWCNet() {
     });
 }
 
+
+//////////////
+// GEMA
+//////////////
+
+function handleGEMA() {
+
+    function findAgencyWorkCodes(tr) {
+        return {
+            'GEMA': [
+                tr.querySelector('.workSocworkcde').innerText.match(/(\d{0,8})[\-‐](\d{3})/)[0],
```

Comment:
That's copied verbatim from the [validation library](https://github.com/ROpdebee/mb-userscripts/blob/d27189dfd408547ed8f5ab07f08a8fc17e317f10/lib/work_identifiers.js) as it currently doesn't expose the agency patterns.
I don't think it would be worth the effort, given that the pattern is only a fancy way to strip the "GEMA work no.:" or "GEMA-Werk.-Nr:" prefix.
But I'm wondering if the code shouldn't have at least one digit before the version code?
```suggestion
                tr.querySelector('.workSocworkcde').innerText.match(/(\d{1,8})[\-‐](\d{3})/)[0],
```

Reply:
Probably yes, not sure why it allows an empty code...